sunpy/tests/helpers.py

```diff
@@ -105,3 +107,55 @@ def wrapper(*args, **kwargs):
                                'New image generated and placed at {}'.format(result_image_loc))
 
     return wrapper
+
+
+@pytest.mark.remote_data
+def download_baseline_images(figure_base_dir, baseline_fig_dir):
```

Comment:
do we actually need to download these? can we not just link to the ones hosted on github?

Reply:
🤦‍♂️ oh yeah that's how the internet works